scripts/build.js

```diff
@@ -72,6 +73,20 @@ async function main() {
   args.push(src);
   await execa(process.execPath, args, { stdio: 'inherit' });
 
+  await new Promise((res) => {
+    lineReplace({
+      file: join(dirRoot, 'dist/index.js'),
+      line: 1,
+      text: '#!/usr/bin/env node --no-warnings',
```

Comment:
I'm not sure I understand the purpose of dynamically replacing this instead of changing `index.js` ❓

Reply:
@styfle amazing, I thought this was added by `ncc` and didn't even bother to look at `src/index.ts` 🤦‍♂ 

Thanks, replaced it there and reverted those changes